src/components/cards/NFTOffersCard/Offer.tsx

```diff
@@ -41,34 +50,36 @@ export const FakeOffer = () => {
   const { colors } = useTheme();
   return (
     <AccentColorProvider color={colors.skeleton}>
-      <Box
-        background="accent"
-        width={{ custom: NFT_IMAGE_SIZE }}
-        height={{ custom: NFT_IMAGE_SIZE }}
-        borderRadius={12}
-      />
-      <Box paddingBottom={{ custom: 7 }} paddingTop={{ custom: 12 }}>
-        <Inline space="4px" alignVertical="center">
-          <Box
-            background="accent"
-            width={{ custom: 12 }}
-            height={{ custom: 12 }}
-            borderRadius={6}
-          />
-          <Box
-            background="accent"
-            width={{ custom: 50 }}
-            height={{ custom: 9.3333 }}
-            borderRadius={9.3333 / 2}
-          />
-        </Inline>
-      </Box>
-      <Box
-        background="accent"
-        width={{ custom: 50 }}
-        height={{ custom: 9.3333 }}
-        borderRadius={9.3333 / 2}
-      />
+      <Inset vertical="10px" horizontal={{ custom: 7 }}>
```

Comment:
just added an inset here so it plays well with `FlashList`. otherwise shadows get clipped